scripts/mobility-database-harvester/harvest_latest_versions.py

```diff
@@ -145,23 +51,29 @@ def save_content_to_file(content, data_path, filename):
         json.dump(content, f)
```

Comment:
Instead of writing content to a file with a hard-coded name, I'd consider just writing to stdout directly and capturing that output directly in the workflow script, especially if you don't actually need the jq step.  If that doesn't work for some reason, I'd still inline the function and consider having the script accept a full path for the output file instead of a directory.  It wasn't quite clear in the workflow script where the `latest_versions.json` file was coming from since it hadn't been specified as an argument to the script.

Reply:
I agree, it makes more sense.